### PR TITLE
Carousel

### PR DESCRIPTION
**Pull request by johnang-del**

### CODE REVIEW

**Comment by johnang-del** [comment]
Screenshot

Screen Shot 2019-09-19 at 10 57 16 AM

**Comment by johnang-del** [comment]
added option for body content on carousel images

Screen Shot 2019-09-19 at 12 24 09 PM

File: src/styles/_carousel.scss [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	}
+	&--back {
+		left: 20px;
+	}
+}
+
+.fa-chevron-right:before,
+.fa-chevron-left:before {
+	color: #fff;
+	opacity: 1;
```
Comment (pauvalenzuela) [inline src/styles/_carousel.scss]: You can remove this opacity. Is there any reason you're using opacity on this one?

Comment (johnang-del) [reply src/styles/_carousel.scss]: left over testing opacity inheritance from parent, this has been removed on my latest commit

**Pull request merged**